Terminal.Gui/Input/Key.cs

```diff
@@ -259,7 +259,7 @@ public Key (string str)
     /// </remarks>
     public static bool GetIsKeyCodeAtoZ (KeyCode keyCode)
     {
-        if (((keyCode & KeyCode.AltMask) != 0) || ((keyCode & KeyCode.CtrlMask) != 0))
+        if ((keyCode & KeyCode.AltMask) != 0 || (keyCode & KeyCode.CtrlMask) != 0)
```

Comment:
These can be combined into a single comparison against the bitwise or of the flags to save several operations and stack churn.